### DISCUSSION TITLE
Section Cups and Support Types ?

### DISCUSSION THREAD

**Discussion by SheepDomination** [sn4k3/UVtools#220] [Q&A]
How can I view suction cups, if any exist ? 

And is it possible to view where the support should change, as in; where the print could collapse or fail due to the support type ?

**Comment by sn4k3** [comment]
A closed area or large areas will always be a suction cup. Thats why most people say to print at an angle with few contact as possible with the plate.
Only hollowed areas with proper vent/drain holes are "not".
Those areas can be identified with the **Outline -> Hollow areas** but it will not check for proper vent/drain hole

![image](https://user-images.githubusercontent.com/113281/122087463-a514f980-cdfc-11eb-98af-edb597c31611.png)

Plus if you turn on Edge detection, it will help you visualize the wall mass and get rid of white blob confusion:

![image](https://user-images.githubusercontent.com/113281/122087931-2ec4c700-cdfd-11eb-9a94-a29735aef0a5.png)

UVtools do not identify support types nor know if a pixel is supports or model
